src/App.tsx

```diff
@@ -20,12 +26,18 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(e) => {
+                  const { value } = e.target;
+
+                  setQuery(value);
+                }}
```

Comment:
bad practise to make anonymous arrow functions in JSX.
make function in component and pass reference here